### ISSUE TITLE
[CI] vagrantCentos6#packagingTest UPGRADE fail

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch#25295]
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+5.x+packaging-tests/471/console

```
# Trying to connect to elasticsearch and wait for expected status yellow for index library
# {
#   "cluster_name" : "elasticsearch",
#   "status" : "red",
#   "timed_out" : true,
#   "number_of_nodes" : 1,
#   "number_of_data_nodes" : 1,
#   "active_primary_shards" : 0,
#   "active_shards" : 0,
#   "relocating_shards" : 0,
#   "initializing_shards" : 0,
#   "unassigned_shards" : 0,
#   "delayed_unassigned_shards" : 0,
#   "number_of_pending_tasks" : 0,
#   "number_of_in_flight_fetch" : 0,
#   "task_max_waiting_in_queue_millis" : 0,
#   "active_shards_percent_as_number" : 100.0
# }
# Connected
# Checking that the cluster health matches the waited for status...
# unexpected status:  'green' wanted 'yellow'
ok 242 [UPGRADE] check elasticsearch version is version under test
not ok 243 [UPGRADE] verify that the documents are there after restart
not ok 243 [UPGRADE] verify that the documents are there after restart
# (in test file /project/build/bats/tests/80_upgrade.bats, line 105)
# (in test file /project/build/bats/tests/80_upgrade.bats, line 105)
#   `curl -s localhost:9200/library/book/1?pretty | grep Elasticsearch' failed
#   `curl -s localhost:9200/library/book/1?pretty | grep Elasticsearch' failed
ok 244 [UPGRADE] stop version under test
```

I ran this locally and it also fails on my machine, output looks similar to the above:
```
REPRODUCE WITH: gradle :qa:vagrant:vagrantCentos6#packagingTest -Dtests.seed=CA1448916E7BC726 
```

**Comment by nik9000** [comment]
Looks like we're not finding the rpm or deb. I'll dig into it. I probably broke it, knowing my luck.

**Comment by nik9000** [comment]
Copy and paste errors get the best of us. 929194ef05113b48e2b4183355f894d14fff1584

**Comment by nik9000** [comment]
I forgot to add `Closes #25295` so here are links:

5.x: 52a141c79049836db0bf5af73d95c506a1eca4a8
5.5: 8ec148dc2146c087c34aef3815e3cb9672d97fd0
